json_object.c

```diff
@@ -510,7 +517,8 @@ json_bool json_object_object_get_ex(const struct json_object* jso, const char *k
 
 void json_object_object_del(struct json_object* jso, const char *key)
 {
-	lh_table_delete(jso->o.c_object, key);
+	if(jso)
+		lh_table_delete(jso->o.c_object, key);
```

Comment:
If you dig deeper, there may be a problem if `key` is NULL and the supplied hash functions are not NULL-safe due to this line

    unsigned long h = t->hash_fn(k);

in `lh_table_lookup_entry`